wrapanapi/systems/openstack.py

```diff
@@ -547,17 +549,18 @@ class OpenstackSystem(System, VmMixin, TemplateMixin):
     can_pause = True
 
     def __init__(self, **kwargs):
+        self.keystone_version = kwargs.get('keystone_version', 2)
+        if int(self.keystone_version) not in (2, 3):
+            raise KeystoneVersionNotSupported(self.keystone_version)
```

Comment:
Way to fail fast ;)

Reply:
![image](https://user-images.githubusercontent.com/21315076/42041118-37de7adc-7abf-11e8-8743-2cc3501f98e8.png)